### PR TITLE
Remove 'Restricted denominations' functionality.

### PR DESCRIPTION
**Pull request by haasted**
Issue #56.

### CODE REVIEW

File: x/market/keeper/keeper.go [hunk 1]
```diff
@@ -311,20 +306,14 @@ func (k *Keeper) NewOrderSingle(ctx sdk.Context, aggressiveOrder types.Order) (*
 	} else {
 		// Check whether this denomination is restricted and thus cannot create passive orders
```
Comment (blewater) [inline x/market/keeper/keeper.go:307]: may adjust/remove the comment :)

File: Makefile [hunk 2]
```diff
@@ -82,7 +82,7 @@ test:
 	go test -mod=readonly ./...
 
 bdd-test:
```
Comment (blewater) [inline Makefile:84]: Interesting, it takes ~7 minutes in mine--maybe closer to 30 minutes is sufficient?

**Review by blewater — APPROVED** [review]

**Comment by blewater** [comment]
Future suggestion, now that the proxykeeper has been simplified, another name closer to its mission would help identifying it.

File: x/market/keeper/keeper.go (hunk 1 above)
Comment (haasted) [reply x/market/keeper/keeper.go:307]: Good catch! 👍

**Pull request merged**